src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx

```diff
@@ -0,0 +1,97 @@
+
+[Learn more about the Slack integration](https://docs-preview.newrelic.com/docs/collaboration-slack-integration).
+
+## Built-in screenshot sharing [#built-in-screenshots]
+
+With a new integrated screenshot selection tool you can attach screenshots to your comments. Click the “screenshot” icon, select the portion of the screen you want, and submit your comment. 
+
+## Close and re-open discussions [#close-reopen]
+
+Once a question is answered or issue is resolved you can mark a discussion as “Closed” via the more actions menu on the discussion thread. Similar to a Jira ticket your team can still add comments to the discussion. This will not automatically re-open a discussion but if you need to you can manually do so the same way as closing.
+
+<img
+  title="Close and reopen discussions "
+  alt="A screenshot of the UI that depicts closing discussions."
+  src={close_and_reopen_discussions}
+/>
+
+## Follow and unfollow discussions [#follow-unfollow]
+
+When you add a comment to a discussion or if you're at-mention you'll automatically be set to follow the discussion. This will be indicated by the bell icon on the discussion and on the discussions board. Following a discussion means you'll get email notifications and the Discussions board UMI (see next section) will increment when new comments are added to a discussion. If you want to unfollow a discussion that's also available via the more actions menu on the discussion (see screenshot in doc above).  Likewise, if there is a discussion you are interested in that you're not following but would like to, you can manually follow the same way. 
+
+## The discussions board [#discussions-board]
+
+<img
+  title="Discussions board"
+  alt="A screenshot of the UI that depiects the discussions board."
+  src={collaboration_discussion_board_new}
+/>
+
+Go to the discussions board to see a complete history of the discussion threads in your organization. On the left nav, the UMI on the Discussions link indicates discussion threads you are following that have unread comments.. On the Discussions page, click a discussion thread to jump to the UI where that conversation is happening.
+
+We know that your team uses a wide variety of tools to communicate but when discussions are spread across platforms and teams, it is easy to lose institutional knowledge. You can now use the discussion board to find active discussions as well as search the archive of shared knowledge. Our discussion board helps you find answers to questions and research how issues and problems were solved in the past.
```

Comment:
I like feel and tone of this paragraph a lot. I think there should be a comma after communicate and it is perfect.

Reply:
:)